ethpm/utils/package_validation.py

```diff
@@ -48,20 +49,17 @@ def validate_package_deployments(package_data):
     if set(("contract_types", "deployments")).issubset(package_data):
         all_contract_types = list(package_data["contract_types"].keys())
 
-        deployments = [
-            deployment
-            for uri, deployment
-            in package_data["deployments"].items()
-        ]
-        deployment_names = [
-            name
-            for name, value
-            in deployments[0].items()
-        ]
-
-        if not deployment_names <= all_contract_types:
+        all_deployments = list(package_data["deployments"].items())
```

Comment:
Change `items` to `values()` and you can remove the `[1]` from the next comprehension.